mmv1/third_party/terraform/acctest/vcr_utils.go

```diff
@@ -164,6 +173,31 @@ func VcrTest(t *testing.T, c resource.TestCase) {
 	c.Steps = steps
 
 	resource.Test(t, c)
+
+}
+
+func createFilesForReleaseDiffTest() (*os.File, *os.File, *os.File) {
```

Comment:
I would consider tweaking this function, since returning 3 file handles is a bit unusual and non-obvious. Do you actually need to create the failure files upfront, or could that be done in `writeOutputFileDeferFunction`?